src/Core/Domain/Customer/Group/Command/AddCustomerGroupCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Domain\Customer\Group\Command;
+
+use PrestaShop\Decimal\DecimalNumber;
+
+class AddCustomerGroupCommand
+{
+    /**
+     * @var string[]
+     */
+    private $localizedNames;
+
+    /**
+     * @var DecimalNumber
+     */
+    private $reduction;
+
+    /**
+     * @var bool
+     */
+    private $displayPriceTaxExcluded;
+
+    /**
+     * @var bool
+     */
+    private $showPrice;
+
+    /**
+     * @param string[] $localizedNames
+     * @param DecimalNumber $reduction
+     * @param bool $displayPriceTaxExcluded
+     * @param bool $showPrice
+     */
+    public function __construct(
+        array $localizedNames,
+        DecimalNumber $reduction,
```

Comment:
can it be lower/greater than zero? if there are some of these constraints we could assert it directly in command and "fail fast".